docs/domains.rst

```diff
@@ -62,6 +70,18 @@ Field details:
     characters as well as hyphens ``-`` and underscores ``_`` (except at the
     beginning of the name).  The maximum length is 191.
 
+``published``
+    :Access mode: read-only
+
+    Timestamp of when the domain's DNS records have lasted been published,
+    in ISO 8601 format (e.g. ``2013-01-29T12:34:56.000000Z``).
+
+    As we publish record modifications immediately, this usually indicates
```

Comment:
remove "usually" (the below-mentioned exception is IMHO not important enough to mention it twice)